diagnostic.md

```diff
@@ -36,13 +47,24 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request GET "http://example.com/donuts/6"
 ```
 
 Write an AJAX request to retrieve a **single donut** from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let getDonuts = $('#donut-id').val();
+
+donutApi.show(getDonuts)
+  .then(donutUi.logResponseBody)
+  .catch(donutUi.logRequestError);
+
+const show = function (id) {
+  return $.ajax({
+    url: app.host + '/donuts/' + id,
+    method: 'GET',
+  });
+};
 ```
```

Comment:
Similarly, here we just need the ajax call. Never do more work than you have to!